README.md

```diff
@@ -1,17 +1,28 @@
 ![StackMaster](/logo.png?raw=true)
 
-StackMaster is a sure-footed way of creating, updating and keeping track of
-Amazon (AWS) CloudFormation stacks.
+StackMaster is a CLI tool to manage CloudFormation stacks, with the following features:
 
-- See the changes you are making to a stack before you apply them
-- Connect stacks
-- Keep secrets secret
-- Customise stacks in different environments
-- Apply descriptive labels to regions
+- Synchronous visibility into stack updates. See exactly what is changing and
+  what will happen before agreeing to apply a change.
+- Dynamic paramter resolvers.
+- Template compiler support for YAML and SparkleFormation.
```

Comment:
Maybe keep the link to sparkle as you had before: `[SparkleFormation](http://www.sparkleformation.io)`